### DISCUSSION TITLE
Can't build project after upgrade bartycrouch - Xcode 13, swift 5

### DISCUSSION THREAD

**Discussion by mkcpb** [Flinesoft/BartyCrouch#236] [Q&A]

**Comment by Jeehut** [comment]
Hi @mkcpb, can you provide more context, please?
* What's the output of `swift --version`?
* What macOS version are you on?

Also, what's the last version of BartyCrouch that still works?
I'm on latest stable Xcode & macOS and it works for me both on Intel & Apple Silicon. So can't reproduce...
